handlers/whatsapp/whatsapp.go

```diff
@@ -1025,6 +1026,10 @@ func sendWhatsAppMsg(rc redis.Conn, msg courier.Msg, sendPath *url.URL, payload
 		return wppID, externalID, []*courier.ChannelLog{log, checkLog, retryLog}, err
 	}
 	externalID, err := getSendWhatsAppMsgId(rr)
+	wppID, err := jsonparser.GetString(rr.Body, "contacts", "[0]", "wa_id")
```

Comment:
shouldn't do this if `err != nil` from line above

Reply:
very important, thanks.